hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -71,6 +77,9 @@
 	private String memo = "";
 	private Id proxy;
 	private int autoAssociationMetadata;
+	private TreeMap<EntityNum, Long> cryptoAllowances = new TreeMap<>();
```

Comment:
Here again I would begin with `Collections.emptyMap()` and only instantiate new `TreeMap`s at the moment we are actually changing allowances.

Not a blocker for this PR imo because the `Account` model isn't used in the most performance-critical operation (`CryptoTransfer`)...and we will probably throttle the `approve/adjustAllowance` pretty low.

Reply:
Fixed . Thanks !